tests/lib/snaps/test-snapd-service-watchdog/bin/direct

```diff
@@ -41,7 +41,7 @@ def main(opts):
     else:
         sleep_time = watchdog_timeout / 2
 
-    logging.info('watchdog notification every %us', sleep_time)
+    logging.info('watchdog notification every %s', sleep_time)
```

Comment:
why this one? It changes it from logging e.g. `... every 30s` to `every 30`

Reply:
Indeed, silly me, this should be: `%ss` now - the issue is really that `%u` gives us `0` for a `0.5` float value.

Reply:
if you know it's going to be less than 10000 you could use `%.4g`